Guides/Migration.md

```diff
@@ -2,6 +2,61 @@
 
 This guide provides details for how to migration between major versions of `IGListKit`.
 
+## From 2.x to 3.x
+
+For details on all changes in IGListKit 2.0.0, please see the [release notes](https://github.com/Instagram/IGListKit/releases/tag/3.0.0).
+
+### IGListBindingSectionController
+
+If you were using `IGListDiff(...)` _inside_ a section controller to compute diffs for cells, we recommend that you start using `IGListBindingSectionController` which wraps this behavior in an elegant and tested API.
+
+### IGListCollectionView removed
+
+You can simply Find & Replace `IGListCollectionView` with `UICollectionView` in your project to fix this.
```

Comment:
this won't work completely. `IGListCollectionView` will match the beginning of IGListCollectionView**Layout**.

need to search `IGListCollectionView *` and `IGListCollectionView alloc]`, etc.